File: output/qwen_2.5_72b_instruct/avatar/direct_translation/python/java/atcoder_ABC132_A.java
import java.util.*;

public class atcoder_ABC132_A {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        String S = input.nextLine();
        Map<Character, Integer> c = new HashMap<>();
        for (char i : S.toCharArray()) {
            c.put(i, c.getOrDefault(i, 0) + 1);
        }
        Set<Character> Sa = new HashSet<>(c.keySet());
        if (Sa.size() != 2) {
            System.out.println("No");
            System.exit(0);
        }
        for (char i : Sa) {
            if (c.get(i) != 2) {
                System.out.println("No");
                System.exit(0);
            }
        }
        System.out.println("Yes");
    }
}

File: dataset/avatar/Python/Code/atcoder_ABC132_A.py
S = input ( )
from collections import Counter
c = Counter ( S )
Sa = set ( list ( S ) )
if len ( Sa ) != 2 :
    print ( "No" )
    exit ( )
for i in Sa :
    if c [ i ] != 2 :
        print ( "No" )
        exit ( )
print ( "Yes" )

